python/graphscope/nx/readwrite/multiline_adjlist.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This file adjlist.py is referred and derived from project NetworkX,
+#
+#  https://github.com/networkx/networkx/blob/master/networkx/readwrite/adjlist.py
+#
+# which has the following license:
+#
+# Copyright (C) 2004-2020, NetworkX Developers
+# Aric Hagberg <hagberg@lanl.gov>
+# Dan Schult <dschult@colgate.edu>
+# Pieter Swart <swart@lanl.gov>
+# All rights reserved.
+#
+# This file is part of NetworkX.
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+#
+
+import networkx.readwrite.multiline_adjlist
+
+from graphscope.nx.utils.compat import import_as_graphscope_nx
+
+import_as_graphscope_nx(networkx.readwrite.multiline_adjlist)
```

Comment:
We use `import_as_graphscope_nx` here that this file is no longer "derived" from networkx.

Reply:
fixed